source/apu.c

```diff
@@ -32,7 +32,7 @@ void S9xDeinitAPU()
    }
 }
 
-uint8_t APUROM [64];
+extern uint8_t APUROM [64];
```

Comment:
Those `extern` are all defined in `globals.c`, so the linker was having issues since the same variable were declared in multiple places. I don't know why it wasn't a issue before.